tools/pot/tests/test_graph.py

```diff
@@ -211,7 +211,10 @@ def test_cutting_fq_layers(_params, tmp_path, models):
     ('scaleshift_fuse', 'dldt'),
     ('scaleshift_no_fuse_1', 'dldt'),
     ('scaleshift_no_fuse_2', 'dldt'),
-    ('matmul_divide_const', 'dldt')
+    ('matmul_divide_const', 'dldt'),
+    ('inception-resnet-v2', 'tf'),
```

Comment:
@IndiraSalyahova, is this XML was taken as is? We cannot redistribute models even from OMZ.